client/app/components/layout/SideNav.jsx

```diff
@@ -1,11 +1,39 @@
 import React from 'react';
 import { Button } from 'react-bootstrap';
+import moment from 'moment';
 
-export default () => (
-  <aside className="sidebar">
-    <Button bsStyle="primary" className="btn-block">Create Event</Button>
-    <Button bsStyle="primary" className="btn-block">Home</Button>
-    <Button bsStyle="primary" className="btn-block">Next Week</Button>
-    <Button bsStyle="primary" className="btn-block">Previous Week</Button>
-  </aside>
-);
+export default class SideNav extends React.Component {
+  static propTypes = {
+    date: React.PropTypes.string
+  };
+
+  render() {
+    return (
+      <aside className="sidebar">
+        <Button bsStyle="primary" className="btn-block">Create Event</Button>
+        <Button href={"/"}
+                bsStyle="primary" className="btn-block">Home</Button>
+        <Button href={this._dateParam(this._nextWeek())}
+                bsStyle="primary" className="btn-block">Next Week</Button>
+        <Button href={this._dateParam(this._previousWeek())}
+                bsStyle="primary" className="btn-block">Previous Week</Button>
+      </aside>
+    );
+  }
+
+  _startingDate() {
+    return this.props.date ? moment(this.props.date) : moment();
```

Comment:
Instead of checking if date prop is present provide a default date prop like this:
```
static defaultProps = {
  date: moment().format(...)
};
```